tests/manual-test-cases/Group5-Functional-Tests/5-4-High-Availability.robot

```diff
@@ -17,6 +17,78 @@ Documentation  Test 5-4 - High Availability
 Resource  ../../resources/Util.robot
 Suite Teardown  Nimbus Cleanup  ${list}
 
+*** Keywords ***
+Check VM Info
+    ${rc}  ${output}=  Run And Return Rc And Output  govc vm.info \*
+    Should Be Equal  ${rc}  0
+    Log To Console  ${output}
```

Comment:
and we have a get vm info in resources vsphere-util already, it works slightly differently but can that not work for you as defined?